### PR TITLE
Onboarding: Adds the updated Welcome View design

### PR DESCRIPTION
**Pull request by emilylaguna**
| 📘 Project: #500 |
|:---:|

This adds the initial welcome view design update with confetti and support the Plus and New account states

## Screenshots

| iPhone SE 14.1 | iPhone 8  - 15.5 | iPhone 14 Pro Max - 16.1 |
|:---:|:---:|:---:|
||||

<details>
  <summary>Video Demo: Plus Confetti</summary>

   https://user-images.githubusercontent.com/793774/201782074-a6e8329d-e474-409d-ba29-444ddea6cc86.mp4
</details>

<details>
  <summary>Video Demo: Normal Confetti</summary>

   https://user-images.githubusercontent.com/793774/201782069-b9167b20-c6ec-4e6f-a12c-78d9a3d94fdf.mp4
</details>

## To test

1. Launch the app
2. Tap on the Podcasts tab
3. Tap on the Create Folder button
4. Tap on the Login button
5. Tap on the Unlock button
6. ✅ Verify the Welcome view appears and [matches the designs](https://www.figma.com/file/PviUP0aiZctOprDuuYRwpb/Onboarding-%26-Upgrade-v2?node-id=406%3A26148&t=eKpw3p6ssfv1EmoO-4)
    1. The title says "Thank you"
    2. The Confetti is plus confetti
    3. The icon in the header is gold
8. ✅ Verify you see the plus confetti  
9. Tap the Done button
10. ✅ Verify the view is dismissed
11. In Xcode, open the PlusCoordinator
12. On Line 13, change `.plus` to `.newAccount`
13. Relaunch the app
14. Once you get to the view, ✅ Verify the view [matches the designs](https://www.figma.com/file/PviUP0aiZctOprDuuYRwpb/Onboarding-%26-Upgrade-v2?node-id=406%3A25546&t=eKpw3p6ssfv1EmoO-4)
    1. The title says "Welcome..."
    2. The Confetti is normal confetti
    3. The icon in the header is not gold
16. Tap the Find my Next Podcast button
17. ✅ Verify the view dismisses and you are brought to the Discover view

## Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: podcasts/Onboarding/LoginCoordinator.swift [hunk 1]
```diff
@@ -5,8 +5,11 @@ class LoginCoordinator {
     var navigationController: UINavigationController? = nil
 
     func loginTapped() {
-        let controller = SyncSigninViewController()
+
+        let controller = PlusCoordinator.make(in: navigationController)
         navigationController?.pushViewController(controller, animated: true)
+//        let controller = SyncSigninViewController()
+//        navigationController?.pushViewController(controller, animated: true)
```
Comment (leandroalonso) [inline podcasts/Onboarding/LoginCoordinator.swift:12]: Shall we remove this comment?

File: podcasts/Onboarding/Plus/PlusCoordinator.swift [hunk 2]
```diff
@@ -7,13 +7,21 @@ class PlusCoordinator: ObservableObject {
...
+        navigationController.pushViewController(controller, animated: true)
 
-        handlePricesLoaded()
+//        // If the prices haven't been loaded yet, load them and wait...
+//        guard IapHelper.shared.hasLoadedProducts else {
+//            listenForPrices()
+//            return
+//        }
+//
+//        handlePricesLoaded()
```
Comment (leandroalonso) [inline podcasts/Onboarding/Plus/PlusCoordinator.swift]: And this?

File: podcasts/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -3389,3 +3389,27 @@
 
 /* When loading the End of Year stats stories fails. */
 "eoy_stories_failed" = "Failed to load stories.";
+
+/* Title of the view displayed after a user successfully upgrades to plus */
+"welcome_plus_title" = "Thank you, now let’s get you listening!";
```
Comment (leandroalonso) [inline podcasts/en.lproj/Localizable.strings]: Should it be "let’s" or "let's"? Not sure if there's any difference between those.

File: podcasts/en.lproj/Localizable.strings [hunk 4]
```diff
@@ -3389,3 +3389,27 @@
...
+
+/* Title of the view displayed after a user successfully creates an account */
+"welcome_new_account_title" = "Welcome, now let’s get you listening!";
```
Comment (leandroalonso) [inline podcasts/en.lproj/Localizable.strings]: Same as before.

**Review by leandroalonso — APPROVED** [review]
Works as described!

[nit] With the theme light contrast there's an issue with the nav bar on the plus screen:

File: podcasts/en.lproj/Localizable.strings (hunk 4 above)
Comment (emilylaguna) [reply podcasts/en.lproj/Localizable.strings]: Done in: d70c81a0b2d46ea9613311ba880ddb196c814e8c

File: podcasts/en.lproj/Localizable.strings (hunk 3 above)
Comment (emilylaguna) [reply podcasts/en.lproj/Localizable.strings]: Done in: d70c81a0b2d46ea9613311ba880ddb196c814e8c

File: podcasts/Onboarding/LoginCoordinator.swift (hunk 1 above)
Comment (emilylaguna) [reply podcasts/Onboarding/LoginCoordinator.swift:12]: Not needed, this logic is temporary and is just for testing purposes.

File: podcasts/Onboarding/Plus/PlusCoordinator.swift (hunk 2 above)
Comment (emilylaguna) [reply podcasts/Onboarding/Plus/PlusCoordinator.swift]: Not needed, this logic is temporary and is just for testing purposes.

**Pull request merged**